src/libraries/Common/src/System/Security/Cryptography/RSAAndroid.cs

```diff
@@ -84,12 +84,11 @@ public override byte[] Decrypt(byte[] data, RSAEncryptionPadding padding)
                 SafeRsaHandle key = GetKey();
 
                 int rsaSize = Interop.AndroidCrypto.RsaSize(key);
-                byte[]? buf = null;
                 Span<byte> destination = default;
+                byte[]? buf = CryptoPool.Rent(rsaSize);
```

Comment:
`byte[]`

Also, can't the finally block below be simplified, at a minimum to change the `buf!` to `buf`?

Reply:
Good eye.  Applied all the suggestions.